src/skmultiflow/data/concept_drift_stream.py

```diff
@@ -35,8 +35,8 @@ class ConceptDriftStream(Stream):
         by `np.random`.
 
     alpha: float (optional, default: 0.0)
-        Angle of change to estimate the width of concept drift change. If set will override the width parameter.
-        Valid values are in the range (0.0, 90.0].
+        Angle of change to estimate the width of concept drift change. If set will override
+        the width parameter. Valid values are in the range (0.0, 90.0].
```

Comment:
Perhaps something worth checking: the default parameter is `0.0`, but according to the documentation this value should not be valid: 

_`"Valid values are in the range (0.0, 90.0]."`_

Reply:
You are right about this @smastelini ! I think it worth double-checking the documentation.  
I made a different PR (#241) to address your comment and make this PR only for linting.